container/container.go

```diff
@@ -199,6 +225,17 @@ func (container *Container) WriteHostConfig() error {
 func (container *Container) GetResourcePath(path string) (string, error) {
 	// IMPORTANT - These are paths on the OS where the daemon is running, hence
 	// any filepath operations must be done in an OS agnostic way.
+
+	if runtime.GOOS == "windows" {
```

Comment:
can we separate this cleaning logic between windows and unix?

For windows:
```go
func cleanResourcePath(path string) string {
    if len(path) >= 2 {
        c := path[0]
        if path[1] == ':' && ('a' <= c && c <= 'z' || 'A' <= c && c <= 'Z') {
            path = path[2:]
        }
    }
    return filepath.Join(string(os.PathSeparator), path)     
}
```

For unix:

```go
func cleanResourcePath(path string) string {
    return filepath.Join(string(os.PathSeparator), path)
}
```

And call this function in like 239: 

```go
cleanPath := cleanResourcePath(path)
```